model/gcp/registry.yaml

```diff
@@ -195,3 +195,28 @@ groups:
         brief: >
           The hostname of a GCE instance. This is the full value of the default or [custom hostname](https://cloud.google.com/compute/docs/instances/custom-hostname-vm).
         examples: ['my-host1234.example.com', 'sample-vm.us-west1-b.c.my-project.internal']
+
+  - id: registry.gcp.gen_ai
+    type: attribute_group
+    display_name: Generative AI Attributes
+    brief: >
+      This documents defines attributes for Google generative AI systems.
+    stability: development
+    attributes:
+      - id: gcp.gen_ai.request
```

Comment:
We need to add `parameters` or similar to make it descriptive. Also do you care about separating request/response options? 

Throwing in some suggestions: `gcp.gen_ai.operation.parameters | options | metadata | config | properties`

Reply:
Done. Renamed to `gcp.gen_ai.operation.config`. Used "config", because this is the name that the Google Gen AI SDK has chosen to use in all languages for this concept.